driver/examples/configs/baroclinic_c384_cpu.yaml

```diff
@@ -72,7 +71,7 @@ dycore_config:
   nwat: 6
   p_fac: 0.1
   rf_cutoff: 800.
-  rf_fast: false
+  rf_fast: true
```

Comment:
Has rf_fast=True been implemented?

Reply:
It has not, but when run with it set to false, it throws an NotImplementedError when tau != 0. Should this instead be removed or tau == 0?

Reply:
You could cherry pick over the not-implemented PR from Florian and remove the options from the yaml config.

Reply:
I think I'd rather change tau than set something in the config that we haven't actually implemented in the model, at least for now. I'll create an issue for it as well though so we can implement a config that's internally consistent.